plugins/modules/vmware_vcenter_settings.py

```diff
@@ -499,6 +505,18 @@ def ensure(self):
             exec("diff_config['after']['snmp_receiver_%s_community'] = snmp_receiver_%s_community" % (n, n))
         result['diff'] = {}
 
+        advanced_settings = self.params['advanced_settings']
```

Comment:
Would it be possible to add an example of advanced settings in EXAMPLES sections?

Reply:
I've added an example and updated the documentation. Is it OK for you?